src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -255,9 +255,7 @@ function MoneyRequestConfirmationList({
 
     const policyCurrency = policy?.outputCurrency ?? PolicyUtils.getPersonalPolicy()?.outputCurrency ?? CONST.CURRENCY.USD;
 
-    const mileageRate = TransactionUtils.isCustomUnitRateIDForP2P(transaction)
-        ? DistanceRequestUtils.getRateForP2P(policyCurrency)
-        : mileageRates?.[customUnitRateID] ?? DistanceRequestUtils.getDefaultMileageRate(policy);
+    const mileageRate = TransactionUtils.isCustomUnitRateIDForP2P(transaction) ? DistanceRequestUtils.getRateForP2P(policyCurrency) : mileageRates?.[customUnitRateID] ?? defaultMileageRate;
```

Comment:
the `defaultMileageRate` is coming from a `withOnyx` selector that does the same thing. The difference I suppose is that this one is pulling the default mileage rate from the real policy, and not trying to pull it from the `policyDraft`